.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -57,10 +57,13 @@ body:
 
                   while (window.isOpen())
                   {
-                      while (const auto event = window.pollEvent())
+                      while (const std::optional event = window.pollEvent())
```

Comment:
As a point of style I'd prefer we continue to use `auto` here and in all other similar instances

Reply:
We're basically doing that, only that we're constraining the type inference to a `std::optional` of some `T`. I think it's worthwhile using `std::optional` here instead of `auto` to clearly suggest that an event might or might not be polled.

I considered two alternatives:

- `auto`: hides too much information, not clear that a `std::optional` is returned;
- `std::optional<sf::Event>`: too verbose and redundant, the information about being an event is already clearly available from the name of the variable `event`.

Using `std::optional` strikes a good balance between conciseness and providing useful information to the reader.

Reply:
Before bringing up the same discussion point in every PR, maybe let's leave it as it is right now. Start a separate discussion and the align the code pieces afterwards.

It's not helpful when someone writes code in one way and someone else then goes and keeps changing it afterwards in their way.

Reply:
That's fair, but there was a reason I made this change specifically in this PR, as it was now unobvious that `pollEvent` returns an optional with plain `auto'.